app/views/fact_values/_fact.html.erb

```diff
@@ -1,8 +1,8 @@
 <tr>
   <% unless params[:host_id] -%>
-      <td>
+      <td class="ellipsis">
         <% if fact.host -%>
-            <%= link_to(trunc_with_tooltip(fact.host), host_facts_path(:host_id => fact.host), :title => _("Show %{host} facts") % { :host => fact.host.name }) %>
+            <%= link_to(fact.host, host_facts_path(:host_id => fact.host), :title => _("Show %{host} facts") % { :host => fact.host.name }) %>
```

Comment:
could the ellipsis thing be improved in future so it does not conflict with other tooltips? this seems pretty limiting to me... I'll remove the title for this column...